packages/blockchain-utils-linking/src/ethereum.ts

```diff
@@ -61,25 +63,48 @@ export class EthereumAuthProvider implements AuthProvider {
     return createLink(did, accountId, this.provider, this.opts)
   }
 
-  async requestCapability(params: OcapParams): Promise<Ocap> {
+  async requestCapability(
+    sessionDID: string,
+    streams: Array<StreamID | string>,
```

Comment:
Distant future feature request: we'll need to have a DSL to specify streams as resources here.

Reply:
For now, giving devs freedom to specify all the SIWE payload as Joel suggested would suffice, IMO.